### PR TITLE
Update running tests documentation

### PR DESCRIPTION
**Pull request by AdRiley**
### Pull Request Description

Update running library tests based on my new learnings.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -483,23 +483,56 @@ Running the tests for the JVM enso components is as simple as running
...
 enso --run test/Base_Tests
 enso --run test/Geo_Tests
 enso --run test/Table_Tests
 ```
 
-Or to run just a single test (e.g., `Duration_Spec.enso`):
+Or to run just a single test file (e.g., `Duration_Spec.enso`):
 
 ```bash
 enso --in-project test/Base_Tests --run test/Base_Tests/src/Data/Time/Duration_Spec.enso
```
Comment (JaroslavTulach) [inline docs/CONTRIBUTING.md]: `--in-project` is no longer needed since #8775

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -483,23 +483,56 @@ Running the tests for the JVM enso components is as simple as running
 
 #### Testing Enso Libraries
 
-To test the libraries that are shipped with Enso you need to first build the
-engine, the easiest way to do so is to run `sbt buildEngineDistribution`. That
+To run the tests inside sbt you can use the following command:
```
Comment (JaroslavTulach) [inline docs/CONTRIBUTING.md:486]: Nice, use-case driven documentation.

**Review by JaroslavTulach — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (GregoryTravis) [inline docs/CONTRIBUTING.md]: Runninng with the VS Code debugger at the command line:
```
JAVA_OPTS='-agentlib:jdwp=transport=dt_socket,server=n,address=5005' enso --run test/Base_Tests/src/Data/Time/Duration_Spec.enso should.normalize
```

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -483,23 +483,56 @@ Running the tests for the JVM enso components is as simple as running
...
+```
+
+The above running options also work when debugging.
+
+Alternatively to run the test outisde of sbt you need to first build the engine,
+the easiest way to do so is to run `sbt buildEngineDistributionNoIndex`. That
 will create a distribution in the directory `built-distribution`. The engine
 runner that can be used for running the tests is located at
 `built-distribution/enso-engine-<VERSION>-linux-amd64/enso-<VERSION>/bin/enso`
 (or `enso.bat` for Windows).
```
Comment (GregoryTravis) [inline docs/CONTRIBUTING.md]: On ARM mac:
```
built-distribution/enso-engine-<VERSION>-macos-aarch64/enso-<VERSION>/bin/enso
```

**Review by GregoryTravis — APPROVED** [review]

**Pull request merged**